src/schema/nmdc.yaml

```diff
@@ -617,6 +618,15 @@ classes:
     class_uri: prov:Association
 
 enums:
+  biosample_category_enum:
+    description: Categories that a biosmaple can belong to, whether funding-based, methods-based, etc.
+    permissible_values:
```

Comment:
Should the full names be included in the schema? If so: 
LTER: National Science Foundation's Long Term Ecological Research Network (https://lternet.edu)
SFA: Department of Energy Office of Science Biological and Environmental Research Program Laboratory Science Focus Areas (https://science.osti.gov/ber/funding-opportunities/laboratory-scientific-focus-area-guidance)
NEON: National Science Foundation's National Ecological Observatory Network (https://www.neonscience.org)
FICUS: Facilities Integrating Collaborations for User Science (https://jgi.doe.gov/user-programs/program-info/ficus-overview) 

For now, I would remove NGEE since there is not a large number of these projects, and also WARMING doesn't make sense to me. I would imagine there will be more down the road. Thanks!

Reply:
Great, these expansions were exactly what I was hoping for.

Reply:
I think I got all of your ideas in, except for the SIP expansion and URL. Let me know if you can't see what I'm taking about on the Files Changed tab.